src/style.css

```diff
@@ -1,3 +1,76 @@
+:root {
+  --link-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  line-height: 1;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.logo img {
+  width: 100%;
+  height: auto;
```

Comment:
Setting the width of the logo image to 100% might cause it to stretch if the container size changes. Consider setting a max-width instead to maintain the aspect ratio.